python/ray/autoscaler/kuberay/ray-cluster.complete.yaml

```diff
@@ -97,8 +104,21 @@ spec:
         - name: autoscaler
           image: rayproject/kuberay-autoscaler
           imagePullPolicy: Always
-          command: ["/bin/sh"]
-          args: ["-c", "$HOME/anaconda3/bin/python /home/ray/run_autoscaler_with_retries.py --redis-password 5241590000000000"]
+          env:
+          - name: RAY_CLUSTER_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: RAY_CLUSTER_NAME
+            # This value must match the metadata.name of the RayCluster CR.
+            # The autoscaler uses this env variable to determine which Ray CR to interact with.
+            # TODO: Match with CR name automatically via operator, Helm, and/or Kustomize.
+            value: raycluster-complete
+          command: ["/bin/sh", "-c"]
```

Comment:
Would it be possible to do
```
command: /home/ray/anaconda3/bin/python
args:
- "/home/ray/run_autoscaler_with_retries.py"
- "--cluster-name"
- "$(RAY_CLUSTER_NAME)"
- "--cluster-namespace"
- "$(RAY_CLUSTER_NAMESPACE)"
- "--redis-password"
- "5241590000000000"
```
and avoid the additional shell? See also https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#use-environment-variables-to-define-arguments

Reply:
oh great point, let's do that